cmake/RunTests.cmake

```diff
@@ -12,6 +12,12 @@ set(ENV{XDG_CONFIG_HOME} ${BUILD_DIR}/Xtest_xdg/config)
 set(ENV{XDG_DATA_HOME} ${BUILD_DIR}/Xtest_xdg/share)
 unset(ENV{XDG_DATA_DIRS})
 
+# Test being run inside neovim :terminal unset inherited environment variables.
+if(DEFINED ENV{NVIM})
+  unset(ENV{NVIM_LOG_FILE})
```

Comment:
Shouldn't this be checked/cleared independently?

Reply:
I was going for: *we are inside nvim terminal remove variables that it sets by default*. Mainly because we don't know if it was intentionally set by the user.